modules/react/popup/stories/examples/FocusRedirect.tsx

```diff
@@ -34,28 +34,30 @@ export const FocusRedirect = () => {
   }, [model.state.stackRef, visible]);
 
   return (
-    <Popup model={model}>
-      <HStack spacing="s">
-        <Popup.Target as={DeleteButton}>Delete Item</Popup.Target>
-        <div aria-owns={popupId} style={{position: 'absolute'}} />
-        <Popup.Popper>
-          <Popup.Card width={400} padding="s">
-            <Popup.CloseIcon aria-label="Close" />
-            <Popup.Heading>Delete Item</Popup.Heading>
-            <Popup.Body>
-              <p>Are you sure you'd like to delete the item titled 'My Item'?</p>
-            </Popup.Body>
-            <HStack spacing="s">
-              <Popup.CloseButton as={DeleteButton} onClick={handleDelete}>
-                Delete
-              </Popup.CloseButton>
-              <Popup.CloseButton>Cancel</Popup.CloseButton>
-            </HStack>
-          </Popup.Card>
-        </Popup.Popper>
-        <SecondaryButton>Next Focusable Button</SecondaryButton>
-        <SecondaryButton>Focusable Button After Popup</SecondaryButton>
-      </HStack>
-    </Popup>
+    <Popup.Card model={model}>
```

Comment:
@NicholasBoll Looks like an issue with the popup codemod from #1029. There are also changes in  some other testing stories, `PopupCard.tsx` and modal testing stories.

Reply:
@alanbsmith can you just revert these changes? He's going to push a fix